io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/writer/XLSInstanceWriter.java

```diff
@@ -73,86 +74,118 @@ protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 		boolean solveNestedProperties = getParameter(
 				InstanceTableIOConstants.SOLVE_NESTED_PROPERTIES).as(Boolean.class, false);
 
+		boolean emptyFeatureTypes = getParameter(InstanceTableIOConstants.EXPORT_EMPTY_FEATURETYPES)
```

Comment:
The newly introduce parameter should be defined and documented in the extension, e.g. similar to here:
https://github.com/halestudio/hale/blob/5d335696fd7b8f7e2114d3423ea5d2cacf2445f2/io/plugins/eu.esdihumboldt.hale.io.xls/plugin.xml#L112

Reply:
By the way, the easiest way to add/edit the extensions is using the dedicated editor in Eclipse instead of changing the XML directly. That is also the least error prone.